src/widgets/category/title/search-title-view.js

```diff
@@ -48,7 +48,7 @@ module.exports = cdb.core.View.extend({
         sourceType: analyses.title(sourceType),
         showSource: this.model.get('show_source') && letter !== '',
         sourceColor: sourceColor,
-        layerName: layerName,
+        layerName: _.escape(layerName),
```

Comment:
Should we wrap this in a util function like we do in cartodb? so in the future if we change how we sanitise thins we will have to change it only in one place